src/reducers/PaymentReducer.js

```diff
@@ -15,7 +15,7 @@ const initialState = () => ({
   transaction: null,
   patient: null,
   paymentAmount: currency(0),
-  insurancePolicy: null,
+  discountRate: 0,
```

Comment:
much better. I wonder if we can take it even further and not have the redundant `discountRate` at all.. probably not!